purebred.cabal

```diff
@@ -130,20 +130,21 @@ test-suite unittests
 test-suite uat
   type:                exitcode-stdio-1.0
   hs-source-dirs:      test
-  ghc-options:         -Wall
+  ghc-options:         -Wall -threaded
   main-is:             TestUserAcceptance.hs
   default-language:    Haskell2010
   build-depends:       base
                      , purebred-email
                      , tasty
                      , tasty-hunit
                      , directory
-                     , process
+                     , typed-process
                      , bytestring
                      , ini
                      , temporary
                      , text
-                     , regex-posix
+                     , text-icu
```

Comment:
why is it necessary to change from regex-posix to text-icu?

Reply:
Because I had the plan to change from `String` to `Text`. Hm... Maybe it's a bit overkill...

Reply:
Yeah, let's not do that as part of this effort, if it can possibly be avoided.